blob/src/main/java/io/crate/blob/RemoteDigestBlob.java

```diff
@@ -85,7 +85,7 @@ public static Status fromId(byte id) {
 
     public RemoteDigestBlob(BlobService blobService, String index, String digest) {
         this.digest = digest;
-        this.client = blobService.getInjector().getInstance(Client.class);
+        this.client = blobService.getClient();
```

Comment:
Would be possible to change the constructor argument to take the Client directly